cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -43,7 +43,6 @@ namespace cob_cartesian_controller
 
     struct MoveLinStruct
```

Comment:
General question: struct is in C++ class like: Shouldn't the members end with "_" ?
(Same for twist_controller and obstacle_distance packages)